src/System.Runtime.Extensions/tests/Performance/System/Perf.Convert.cs

```diff
@@ -0,0 +1,1982 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System
+{
+    public class Perf_Convert
+    {
+        [Benchmark(InnerIterationCount = 100000)]
+        [InlineData(1024, 1408, Base64FormattingOptions.InsertLineBreaks)]
+        [InlineData(1024, 1368, Base64FormattingOptions.None)]
+        public void ToBase64CharArray(int binaryDataSize, int arraySize, Base64FormattingOptions formattingOptions)
+        {
+            int innerIterationCount = (int)Benchmark.InnerIterationCount;
+            byte[] binaryData = new byte[binaryDataSize];
+            char[] base64CharArray = new char[arraySize];
```

Comment:
It shouldn't be necessary to pass in the array size.  You could, for example, here call ToBase64String to get the resulting length and then allocate the target array once you know how big it should be.  That should help make the test more robust in the face of getting the input arraySize wrong.